### PR TITLE
Fixed up header styles for mobile

### LINKED ISSUE #1529: Register button cut off on mobile

### PR DESCRIPTION
**Pull request by local-girl**
Fixes #1529

### CODE REVIEW

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_moving_columns.scss [hunk 1]
```diff
@@ -155,8 +155,8 @@ The menu should always have the same state as its columns.
 }
 
 %moving-column-single-width {
-    max-width: $font-size-normal * 55;
-    min-width: $font-size-normal * 35;
+    max-width: $moving-column-single-width-max;
+    min-width: $moving-column-single-width-min;
 }
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_moving_columns.scss]: This is not used anymore and should be removed

**Comment by xi** [comment]
I checked again and saw that `minShowWidth` in the JavaScript code is only used to define the breakpoint on which we switch from two-column to one-column mode. It is not enforced as a minimal column width. So that's why it is perfectly safe to remove that rule from CSS. Great!

File: src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_layout.scss [hunk 2]
```diff
@@ -201,6 +201,12 @@ depends on a lot of factors:
 
 .l-header-right {
     float: right;
+
+    @media (max-width: $moving-column-single-width-min) {
+        i[class*=icon-] {
+            display: none;
+        }
+    }
```
Comment (xi) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_layout.scss]: Why this change? I see no real benefit in hiding the icon.

Comment (local-girl) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_layout.scss]: Just thought it gives a bit more breathing room

Comment (local-girl) [inline src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/_layout.scss]: minShowWidth is used in css kind of. When A3 was designed - when you have 2 columns the header width remains a 1 column width as opposed to stretching out, this would look weird on a wide monitor

**Comment by xi** [comment]
LGTM.
